pages/common/gcloud.md

```diff
@@ -0,0 +1,32 @@
+# gcloud
+
+> The official CLI tool for Google Cloud Platform.
+> You can use this tool to perform common platform tasks.
```

Comment:
I wonder if this line is needed. It does not say anything extra except the obvious. The first line already mentions this is the tool for Google Cloud Platform, so it is clear to me that it will be used to perform platform tasks.

Reply:
SGTM